corehq/apps/hqwebapp/templates/hqwebapp/partials/global_navigation_bar.html

```diff
@@ -31,7 +32,13 @@
             {% else %}
                 <a class="btn btn-project{% if is_project_settings %} btn-project-active{% endif %}"
                    href="{% url "domain_settings_default" domain %}">
+                    {% if request|tzmigration_status == 'complete' %}
+                    <i class="icon-globe" title="New timezone behavior enabled"></i>
+                    {% elif request|tzmigration_status == 'in_progress' %}
+                    <i class="icon-globe" title="Timezone migration started but not complete"></i>
+                    {% else %}
                     <i class="icon-cog"></i>
+                    {% endif %}
```

Comment:
Should this also be added to the [BS3 version](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/templates/style/includes/navigation_global.html#L33)?